src/v0/util/index.js

```diff
@@ -1472,13 +1472,16 @@ const getErrorStatusCode = (error, defaultStatusCode = HTTP_STATUS_CODES.INTERNA
 /**
  * Used for generating error response with stats from native and built errors
  */
-function generateErrorObject(error, defTags = {}, shouldEnrichErrorMessage = false) {
+function generateErrorObject(error, defTags = {}, shouldEnrichErrorMessage = true) {
   let errObject = error;
   let errorMessage = error.message;
+  if (errorMessage?.includes('message')) {
```

Comment:
this could be string as well right? in that case delete will fail right?

Reply:
change it to errorMessage?.message

Reply:
Put in extra condition `typeof errorMessage === 'object' && errorMessage?.message`